src/main/java/co/inventorsoft/academy/springBootTask/repository/impl/TournamentBracketRepositoryImpl.java

```diff
@@ -0,0 +1,89 @@
+package co.inventorsoft.academy.springBootTask.repository.impl;
+
+import co.inventorsoft.academy.springBootTask.domain.entity.TournamentBracket;
+import co.inventorsoft.academy.springBootTask.repository.EntityMapper;
+import co.inventorsoft.academy.springBootTask.repository.TournamentBracketRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Repository;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+@RequiredArgsConstructor
+@Repository
+@Slf4j
+public class TournamentBracketRepositoryImpl implements TournamentBracketRepository {
+
+    private static final String SQL__FIND_ALL_TOURNAMENT_BRACKETS = "SELECT * FROM tournament";
+    private static final String SQL__ADD_TOURNAMENT_BRACKET =
+            "INSERT into tournament(matches, winner) " + "values(?, ?)";
+
+    public List<TournamentBracket> findAll() {
+        List<TournamentBracket> tournamentBracketList = new ArrayList<TournamentBracket>();
+        PreparedStatement pstmt = null;
+        ResultSet rs = null;
+        Connection con = null;
+        try {
+            con = DBManager.getInstance().getConnection();
+            TournamentBracketRepositoryImpl.TournamentBracketMapper mapper =
+                    new TournamentBracketRepositoryImpl.TournamentBracketMapper();
+            pstmt = con.prepareStatement(SQL__FIND_ALL_TOURNAMENT_BRACKETS);
+            rs = pstmt.executeQuery();
+            while (rs.next())
```

Comment:
This piece of code is correct but I must ask you to add brackets.
This way it is always easier to read your code.